mitodl/edx-sga
# BOM-2175 : Upgrade To Python 3.8

#### What are the relevant tickets?

- https://openedx.atlassian.net/browse/BOM-2175

#### What's this PR do?

- Dropped support for Python 2.7 , 3.5 and Django 1.11
- Upgrade To Use Python 3.8 and Django 2.2

#### How should this be manually tested?
Login to https://sgaonpy38.sandbox.edx.org and use edx-sga xblock in any course as a publisher and publish to see changes.
